test/cloud_integration_test.go

```diff
@@ -19,7 +19,7 @@ var _ = Describe(versionedPrefixForQuarantining()+"Cloud Mode Integration Tests"
 	withAndWithoutInheritedEnv(func(getEnv envGenerator, prefix string) {
 		getEnvWithAccessToken := func() map[string]string {
 			env := getEnv()
-			env["CAPTAIN_HOST"] = "staging.captain.build"
+			env["CAPTAIN_HOST"] = "staging.cloud.rwx.com"
```

Comment:
Once we have a version released with _this_ captain host in the integration test, then I think we can re-pin the legacy versions to test to _this_ version.